Articles/Blog/what-is-an-inclusive-minute.md

```diff
@@ -0,0 +1,143 @@
+
+Its useful to consider the appropriate cadence for the inclusive minute practice as well.
+If your team meets once a week, identifying new IM topics for every meeting may become a challenge.
+Doing an IM every other meeting or the first meeting of each month may be best.
+If your team meets monthly, having an inclusive minute with every meeting may make the most sense.
+
+Some teams prefer to have inclusive minute at the end of their meetings and others prefer the beginning.
+There are advantages and disadvantages to either.
+At the beginning, follow-up dialog can run longer than expected putting the meeting leader in the position of having to cut-off conversation to get to the rest of the agenda.
+But, having the exercise at the beginning of the meeting also ensures it actually happens.
+It shows commitment.
+At the end of the meeting, unless the team is committed to reserving time for the activity, it can feel rushed and/or like an afterthought.
+But, having the activity at the end does provide a natural way for any follow-up conversation to adjourn.
+
+It works best if leadership of the practice can regularly rotate to various members of the team.
+That said, no team member should be *required* to lead an inclusive minute.
+Its best if team members are encouraged (perhaps even by offering them assistance in preparing an inclusive minute) to consider leading one but leadership should otherwise be on a voluntary basis.
+
+It may be helpful, early on, to exercise some oversight by checking in with prospective IM leaders and ensuring their chosen topic and plans for sharing fit within scope and time constraints.
+Once the team gains some experience, oversight may become unnecessary.
+Finally, to avoid repeats, it is also worth maintaining a catalog of inclusive minute topics.
+This can be useful for maintaining lists of both IM topics already shared as well as ideas for future topics.
+
+#### Dealing with conflict
+
+In spite of the fact that inclusion and inclusive practices are aimed at increasing access and participation for *everyone*, there will always be some who view it as primarily a gesture of political correctness or an imposition, off topic, and may even act against it by constantly playing a contrarian, or otherwise aiming to derail conversation.
+When such conflicts arise, what can a team leader do?
+
+The first place team leaders can look is their own institution's stated policies and procedures which often include valuing and furthering the goals of inclusion.
+See, for example, those of the [Department of Energy (DOE)](https://www.energy.gov/science/diversity-equity-inclusion) or [Lawrence Livermore National Lab](https://www.llnl.gov/diversity/our-values).
+When team members act in ways to derail conversation or which are otherwise contrary to inclusive practices, they should be reminded that they are then acting in ways contrary to the stated values of their institution.
```

Comment:
```suggestion
When team members act in ways to derail conversation or are otherwise contrary to inclusive practices, they should be reminded that they are then acting against the stated values of their institution.
```